build.sbt

```
@@ -186,3 +186,33 @@ webpack := {
 runner := runner.dependsOn(webpack).value
 dist := dist.dependsOn(webpack).value
 stage := stage.dependsOn(webpack).value
+
+// Docker
+
+enablePlugins(DockerPlugin)
+import com.typesafe.sbt.packager.docker.{Cmd, ExecCmd}
+import com.typesafe.sbt.packager.docker.DockerChmodType
+
+Universal / mappings += file("docker/start.sh") -> "bin/start"
+
+dockerUpdateLatest := true
+dockerBaseImage := "adoptopenjdk/openjdk8:alpine-slim"
+dockerChmodType := DockerChmodType.Custom("u=rwX,g=rX,o-rwx")
+dockerExposedPorts ++= Seq(8090, 8443)
```

This may make more sense as (since it is `8080` for prod):
```suggestion
dockerExposedPorts ++= Seq(8080, 8443)
```